api/subjects/views.py

```diff
@@ -108,7 +108,11 @@ class SubjectList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
     ordering = ('is_other', '-id',)
 
     def get_default_queryset(self):
-        return optimize_subject_query(Subject.objects.all())
+        return optimize_subject_query(
+            Subject.objects.filter(
+                provider=PreprintProvider.get_default(),
```

Comment:
Minor: I understand that this is to acquire "standard" BePress subjects -- would `.filter(bepress_subject__isnull=True)` make more sense? It would be equivalent, one fewer query, and potentially less likely to break in the future, if anything changes.

Near as I can tell, nothing is using this endpoint (only the `*ProviderSubjectList`s), which obviates my other question here.

Reply:
agree `.filter(bepress_subject__isnull=True)` seems better -- updated

it isn't used yet, was eventually meant for [adding subjects to projects](https://openscience.atlassian.net/browse/ENG-4354), so the frontend subject-picker knows what subjects to pick from (and iirc was a shortcut compromise rather than adding `ProjectProvider` or doing a taxonomy-provider divorce)